### PR TITLE
Pin KFDef to v1.0.0 tag

### PR DESCRIPTION
**Pull request by swiftdiaries**

### CODE REVIEW

File: apps/networking/ble-localization/onprem/install/README.md [hunk 1]
```diff
@@ -16,13 +16,7 @@ With sudo access on the UCS machine run:
 	sudo apt install nfs-kernel-server
 	
 ### <a id=run-nfs-installationsh></a> Run nfs-installation.sh 
-
+     export UCS_CLUSTER_IP=$(hostname -I | awk '{print $1}')
```
Comment (prem0912) [inline apps/networking/ble-localization/onprem/install/README.md]: Instead of hostname we can use k8's master IP from kubectl custer-info.

Comment (prem0912) [inline apps/networking/ble-localization/onprem/install/README.md]: what if UCS server has multiple nic and they have to run k8s on specific IP.

**Comment by swiftdiaries** [comment]
Good point. Removed Github Action for later and added just the information to fix the upstream issue on the v1.0-branch for kfdef

File: install/kubeflowup.sh [hunk 2]
```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
-
-KF_APP=kf-app
+export KF_APP=kf-app
+export KFDEF_URL="https://raw.githubusercontent.com/kubeflow/manifests/v1.0.0/kfdef/kfctl_k8s_istio.v1.0.0.yaml"
+export KFCTL_URL="https://github.com/kubeflow/kfctl/releases/download/v1.0/kfctl_v1.0-0-g94c35cf_linux.tar.gz"
 mkdir -p ${KF_APP}
 cd ${KF_APP}
-wget -O kfctl.tar.gz https://github.com/kubeflow/kfctl/releases/download/v1.0/kfctl_v1.0-0-g94c35cf_linux.tar.gz
+wget --no-check-certificate -O kfctl.tar.gz ${KFCTL_URL}
```
Comment (ramdootp) [inline install/kubeflowup.sh]: do we need this option "--no-check-certificate" ?

File: apps/networking/ble-localization/onprem/install/README.md [hunk 3]
```diff
@@ -16,13 +16,10 @@ With sudo access on the UCS machine run:
 	sudo apt install nfs-kernel-server
 	
 ### <a id=run-nfs-installationsh></a> Run nfs-installation.sh 
+     
+     $ sh nfs-installation.sh
+     Provide UCS Cluster IP (ex: 10.10.10.101)
+     UCS Cluster IP:
```
Comment (ramdootp) [inline apps/networking/ble-localization/onprem/install/README.md]: These changes are no longer relevant. you need to discard the changes

Comment (ramdootp) [reply apps/networking/ble-localization/onprem/install/README.md]: It got renamed to INGRESS_IP

File: install/controllerkubeup.sh [hunk 4]
```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 
-if [ -z $ipaddress ];
+if [ -z $UCS_CLUSTER_IP ];
```
Comment (andreyvelich) [inline install/controllerkubeup.sh]: If we rename it to `UCS_CLUSTER_IP`, should we also change README here: https://github.com/CiscoAI/cisco-kubeflow-starter-pack/blob/master/install/k8sup.md#-create-cluster-with-kubeadm

Comment (swiftdiaries) [reply install/controllerkubeup.sh]: Changed to use INGRESS_IP everywhere

File: apps/networking/ble-localization/onprem/install/README.md (hunk 3 above)
Comment (swiftdiaries) [reply apps/networking/ble-localization/onprem/install/README.md]: Changed it

File: install/kubeflowup.sh (hunk 2 above)
Comment (swiftdiaries) [reply install/kubeflowup.sh]: Removed it. Was testing this on GitHub Actions for automated testing and thought I'd try to see if that flag changes things

**Pull request merged**